.env

```diff
@@ -96,7 +100,7 @@ MF_BOOTSTRAP_PORT=8202
 MF_BOOTSTRAP_DB_PORT=5432
 MF_BOOTSTRAP_DB_USER=mainflux
 MF_BOOTSTRAP_DB_PASS=mainflux
-MF_BOOTSTRAP_DB=bootstrap
+MF_BOOTSTRAP_DB_NAME=bootstrap
```

Comment:
Following this, we should use `_DB_NAME` for `THINGS_BD`, `AUTHN_DB`, `USERS_DB`... So, either revert everything back to the `..._DB`, or change everything to `..._DB_NAME`. The first option is simpler and introduces fewer changes. I find both options equally unambiguous, so I'm leaning towards the first one.